src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/RSAOAEPKeyExchangeFormatter.cs

```diff
@@ -47,6 +47,11 @@ public override string Parameters
             get {return null;}
         }
 
+        public RandomNumberGenerator Rng {
+            get { return null; }
+            set { throw new PlatformNotSupportedException(); }
```

Comment:
Why is this returning null / throwing?  Based on a quick look at the desktop implementation, should this instead be:
```C#
public RandomNumberGenerator Rng { get; set; }
```
?